theta_dup/include/theta_sketch_dup.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package datasketches_pb;
+
+message Update_theta_sketch_dup {
```

Comment:
CamelCase - https://developers.google.com/protocol-buffers/docs/style#message_and_field_names

Here and below

Reply:
I didn't notice this, thanks!